.travis.yml

```diff
@@ -131,9 +128,17 @@ jobs:
         script: bash travis/trigger_quip_docker.sh
         on:
           branch: public
-      
-        
-notifications:
- email:
-   - quip-developers@eng.cam.ac.uk
 
+  exclude:
+    - if: type = pull_request
+      env: QUIP_ARCH=linux_x86_64_gfortran QUIP_INC=Makefile.inc GAP=1
```

Comment:
why are you turning GAP on when it's a pull request? I thought your changes are precisely to allow to turn it off in that case.

Reply:
The `exclude` key ignores the `env` defined, so this syntax correctly skips the corresponding build matrix line when the conditional is true. In effect:
- If it is a PR
- **skip** the build matrix line with `env` matching ...

Reply:
ah ok. thanks. Just getting started with editing these scripts..